POEApi.Model.Tests/PoeModelTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.IO;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using POEApi.Transport;
+
+namespace POEApi.Model.Tests
+{
+    [TestClass]
+    public class PoeModelTests
+    {
+        private Mock<ITransport> _mockTransport;

```

Comment:
Nitpick, but please remove the underscores from private variables to maintain naming consistency throughout the app.